[debug] [2025-03-26T11:54:54.366Z] ----------------------------------------------------------------------
[debug] [2025-03-26T11:54:54.369Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\USER\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js projects:create
[debug] [2025-03-26T11:54:54.369Z] CLI Version:   13.35.1
[debug] [2025-03-26T11:54:54.370Z] Platform:      win32
[debug] [2025-03-26T11:54:54.370Z] Node Version:  v22.3.0
[debug] [2025-03-26T11:54:54.370Z] Time:          Wed Mar 26 2025 18:54:54 GMT+0700 (Western Indonesia Time)
[debug] [2025-03-26T11:54:54.370Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-03-26T11:54:54.373Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2025-03-26T11:54:54.422Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-03-26T11:54:54.422Z] > authorizing via signed-in user (stefanie_2226240009@mhs.mdp.ac.id)
[debug] [2025-03-26T11:54:54.993Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2025-03-26T11:54:54.993Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[debug] [2025-03-26T11:56:31.831Z] ----------------------------------------------------------------------
[debug] [2025-03-26T11:56:31.834Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\USER\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js projects:create
[debug] [2025-03-26T11:56:31.834Z] CLI Version:   13.35.1
[debug] [2025-03-26T11:56:31.834Z] Platform:      win32
[debug] [2025-03-26T11:56:31.835Z] Node Version:  v22.3.0
[debug] [2025-03-26T11:56:31.835Z] Time:          Wed Mar 26 2025 18:56:31 GMT+0700 (Western Indonesia Time)
[debug] [2025-03-26T11:56:31.835Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-03-26T11:56:31.843Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-03-26T11:56:31.843Z] > authorizing via signed-in user (stefanie_2226240009@mhs.mdp.ac.id)
[debug] [2025-03-26T11:58:15.180Z] Checked if tokens are valid: false, expires at: 1742554012157
[debug] [2025-03-26T11:58:15.180Z] Checked if tokens are valid: false, expires at: 1742554012157
[debug] [2025-03-26T11:58:15.181Z] > refreshing access token with scopes: []
[debug] [2025-03-26T11:58:15.183Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-03-26T11:58:15.183Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-03-26T11:58:15.668Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-03-26T11:58:15.668Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-03-26T11:58:15.685Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects [none]
[debug] [2025-03-26T11:58:15.685Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects {"projectId":"fasum-apps","name":"fasum-apps"}
[debug] [2025-03-26T11:58:17.036Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects 409
[debug] [2025-03-26T11:58:17.036Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects {"error":{"code":409,"message":"Requested entity already exists","status":"ALREADY_EXISTS"}}
[debug] [2025-03-26T11:58:17.357Z] FirebaseError: Request to https://cloudresourcemanager.googleapis.com/v1/projects had HTTP Error: 409, Requested entity already exists
    at responseToError (C:\Users\USER\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:52:12)
    at RetryOperation._fn (C:\Users\USER\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:312:77)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[error] 
[error] Error: Failed to create project because there is already a project with ID fasum-apps. Please try again with a unique project ID.
[debug] [2025-03-26T11:58:38.470Z] ----------------------------------------------------------------------
[debug] [2025-03-26T11:58:38.472Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\USER\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js projects:create
[debug] [2025-03-26T11:58:38.473Z] CLI Version:   13.35.1
[debug] [2025-03-26T11:58:38.473Z] Platform:      win32
[debug] [2025-03-26T11:58:38.473Z] Node Version:  v22.3.0
[debug] [2025-03-26T11:58:38.473Z] Time:          Wed Mar 26 2025 18:58:38 GMT+0700 (Western Indonesia Time)
[debug] [2025-03-26T11:58:38.473Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-03-26T11:58:38.483Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-03-26T11:58:38.484Z] > authorizing via signed-in user (stefanie_2226240009@mhs.mdp.ac.id)
[debug] [2025-03-26T11:58:59.644Z] Checked if tokens are valid: true, expires at: 1742993894668
[debug] [2025-03-26T11:58:59.645Z] Checked if tokens are valid: true, expires at: 1742993894668
[debug] [2025-03-26T11:58:59.647Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects [none]
[debug] [2025-03-26T11:58:59.647Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects {"projectId":"fasum-app","name":"fasum"}
[debug] [2025-03-26T11:59:00.953Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects 409
[debug] [2025-03-26T11:59:00.953Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects {"error":{"code":409,"message":"Requested entity already exists","status":"ALREADY_EXISTS"}}
[debug] [2025-03-26T11:59:01.129Z] FirebaseError: Request to https://cloudresourcemanager.googleapis.com/v1/projects had HTTP Error: 409, Requested entity already exists
    at responseToError (C:\Users\USER\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:52:12)
    at RetryOperation._fn (C:\Users\USER\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:312:77)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[error] 
[error] Error: Failed to create project because there is already a project with ID fasum-app. Please try again with a unique project ID.
